contributing.md

```diff
@@ -191,3 +191,34 @@ Once you're configured, `grunt` tasks are available:
     grunt watch                # watch coffee/ for changes and build Chosen
 
 If you're interested, you can find the task in [Gruntfile.coffee](https://github.com/harvesthq/chosen/blob/master/Gruntfile.coffee).
+
+# Release a new version of Chosen
+
+This documentation is for Chosen maintainers.  You must have write permissions for this repository to cut a release.
+
+1. Bump the version number in the source but _do not push it yet_. [Example commit](https://github.com/harvesthq/chosen/commit/be0a298f528ec59ce97889eaeeeb47a2dca9ca79).
```

Comment:
Why the "do not push it yet"? You can create tags for already pushed commits, without changing history.

Reply:
After inspecting the process, the steps are actually; update `package.json`, run `grunt prep-release` to update the version in the generated files, and only after that a commit should be made.

Reply:
We could actually add the commit and tag steps to the `prep-release` task

Reply:
@koenpunt The "do not push yet" was part of the first issue I commented upon — the Travis build step is:

1. Triggered by a commit hitting master
2. Dependent on the version tag existing

So if you pushed the commit first, the build would happen without the tag and the version would be wrong.

---

As for "update `package.json`, run `grunt prep-release`", that misses the [version string in `package-lock.json`](https://github.com/harvesthq/chosen/blob/be0a298f528ec59ce97889eaeeeb47a2dca9ca79/package-lock.json#L3).

It looks like committing package-lock.json is a recent change: https://github.com/harvesthq/chosen/commit/b0ac854ea0d3771bf3af2796d193bafb25770aef#diff-32607347f8126e6534ebc7ebaec4853d — should we be doing that?

Reply:
But when there's no new tag, the publish script just doesn't do anything, and when then the tag is pushed, the script runs again and publishes the release. Or at least I'm pretty sure I've always released it like that. But I might of course be wrong. 
In that case I think you should add your comment from above to the document too, to clarify why one should not push yet

---

Regarding `package-lock.json`; it can be safely added. 

Updating the lock file is as easy as running `npm install` I think.

But the other day I have been looking at using `npm version`, for automating updating the version in the `package.json`, and can be configured with some hooks to automatically run `grunt prep-release` too.

I'll see if I can have that working today.